components/dash-core-components/src/components/Checklist.react.js

```diff
@@ -108,6 +108,7 @@ Checklist.propTypes = {
                     PropTypes.string,
                     PropTypes.number,
                     PropTypes.bool,
+                    PropTypes.node,
```

Comment:
Should these switch to just `label: PropTypes.node`? Strings and numbers are valid as nodes - as are booleans, though they aren't rendered. And this is already reflected in the docstrings: `node` currently creates the docstring "a list of or a singular dash component, string or number" (ignoring bool, which I think is appropriate), so as is this is shown as:
```
- label (string | number | boolean | a list of or a singular dash component, string or number; required):
    The option's label.
```

It's good to know that `oneOfType` works with `node` though!

Reply:
> a list of or a singular dash component, string or number

In this case, component in a list of dictionary, it cannot be a list of components but a single component. Maybe we can put PropTypes.element instead ?

Reply:
This is still open: rather than adding to the `oneOfType` I think this should be just `label: PropTypes.node`. Note that today `boolean` doesn't work anyway (both `True` and `False` show no label at all) so `node` that already allows `string` and `number` (and based on your other work lists of all of these are now allowed?) should be exactly what we want, right?

Reply:
And same for dropdown and radioitems